### DISCUSSION TITLE
Dependency Track does not generate a report when No finding were retrieved

### DISCUSSION THREAD

**Discussion by loveshjain** [pmckeown/dependency-track-maven-plugin#178] [Q&A] [answered]
Hi,

With the version 0.8.3 release https://github.com/pmckeown/dependency-track-maven-plugin/pull/91/files#diff-ac0a579116298de0a120aa65708a865a38803cb8c60d83c05556f5158d6a90ef, dependency track plugin does not generate the reports when "No findings were retrieved". 

There are occasions in our projects where we get "No findings were retrieved" and was wondering whether it should be generating a report for that use case or not. 

Not sure if it is an issue with dependency-track plugin hence instead of raising an issue, I started a discussion.

Thanks,
Lovesh

**Comment by pmckeown** [comment]
Hi Lovesh, thanks for raising this.  Looks like the issue is still present in the latest version.  

I will raise an issue and try to get this fixed soon.

**Comment by pmckeown** [comment, marked as answer]
The new version 1.0.0 has the fix for this issue in.

Cheers for reporting Lovesh.

Would appreciate it if you could test it out in your set up to see if it works for you.
